main.mk

```diff
@@ -1,5 +1,5 @@
 .PHONY: latest
-latest:: txt html
```

Comment:
I'd prefer it if you leave this line alone.  Building raw by default is an option you can enable by adding

```make
latest:: raw
```

to your `Makefile`.